src/client/rewards/PaymentCard/TransferButton.js

```diff
@@ -25,11 +25,12 @@ const TransferButton = ({
   openTransf,
 }) => {
   const isReceiverGuest = name.startsWith(GUEST_PREFIX) || name.startsWith(BXY_GUEST_PREFIX);
```

Comment:
`re` to check guest
https://github.com/Waiviogit/waivio/blob/b4d4186f3f45cdcbde1b14726f945d16781964d7/src/client/helpers/regexHelpers.js#L20